src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_AMOUNT = 5;
+    private static final int MAX_RANDOM_NUMBER = 35;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_AMOUNT);
+
+        switch (figureNumber) {
+            case 0:
+                return getRandomSquare();
+            case 1:
+                return getRandomRectangle();
+            case 2:
+                return getRandomRightTriangle();
+            case 3:
+                return getRandomCircle();
+            default:
+                return getRandomTrapezoid();
+        }
+    }
+
+    private Square getRandomSquare() {
+        int sideLength = random.nextInt(MAX_RANDOM_NUMBER);
```

Comment:
```suggestion
        int sideLength = getRandomLength();
```
create one more private method